src/components/Board/index.js

```diff
@@ -4,34 +4,39 @@ import Square from '../Square';
 
 
 const Board = (props) => {
+
     
-    const renderSquare = i => {
+    const renderSquare = i => { 
+       
         return (<Square value = {props.squares[i]}
-          onClick={()=> props.onClick(i)}/>
+          onClick={()=> props.onClick(i)} key = {i} bright = {  (props.highlight != null &&  props.highlight.includes(i))  ?  true : false  } />
```

Comment:
A `highlight` prop for Board doesn't make much sense to me. Instead, I think is better to pass the `winnerLine` as a prop so the Board can make calculations with it. Then you can define an `isWinnerSquare` function to decide to highlight a Square or not.

```suggestion
          onClick={()=> props.onClick(i)} key = {i} bright = {isWinnerSquare(i)} />
```

```
const isWinnerSquare = (i) => winnerLine && winnerLine.includes(i)
```